<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node pkg="ipcamera_driver" type="ipcamera_driver" name="limelight" output="screen">
        <param name="video_url" type="string" value="http://10.0.88.42:5802/" />
        <!-- <param name="camera_info_url" type="string" value="package://mrl_ipcamera/launch/example_parameters.yaml" /> -->
        <param name="frame_id" type="string" value="limelight_link" />
        <remap from="camera/image" to="limelight/image_raw" />
    </node>
</launch>
